### ISSUE TITLE
Inline Editable - Avoid use of placeholder values to label or explain input - (2036569913)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5574]
### Module: 
 15 Inline Editable 

### Violation: 
 Avoid use of placeholder values to label or explain input 

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036569913 

### Severity: 
 8 

### Description: 
 [Issue]
There are form fields that rely on the placeholder attribute to supply their accessible name. Examples include:

- Inline Editable
- Inline Editable with Controls
- Inline Editable with Loading Controls

[User Impact]
Users with cognitive disabilities may be unable to determine the purpose of the field after the placeholder disappears. Some assistive technologies will not use the placeholder attribute as a field&#039;s accessible name, so screen reader users will have difficulty determining what fields are for and speech input users will have difficulty navigating to them.

[Code Reference] 
 ```
<input aria-label="" class="inline-child" enterkeyhint="" inputmode="" placeholder="Inline Editable" tabindex="-1" type="text">
 ``` 

### Note: 
 [Suggestion]
Avoid use of placeholder values to label or explain input.

Labels must meet the following requirements:
- The label must be visible.
- The label must be persistent. That is, a label must not disappear when text is entered into the field or an option is selected.
- The label must be programmatically associated with the form field. The most common way to do this is with a ``` <label> ``` element with a for attribute set to the ID of the field.
- The label must provide the accessible name of the field, or the label&#039;s exact text must be included in the accessible name.

[Compliant Code Example] 
 ```
<label for="A1"> Inline Editable</label>
<input class="inline-child" enterkeyhint="" inputmode="" placeholder="Inline Editable" tabindex="-1" type="text" id="A1">
 ```
 

### Media Type: 
 Forms 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This is a doc-related update. Closing and adding context to doc updates for `input` in its component page and when used in `inline-editable`.
